cmd/admin-info.go

```diff
@@ -187,31 +187,35 @@ func mainAdminInfo(ctx *cli.Context) error {
 	// If the error is not nil and not unrecognizable, just print it and exit
 	fatalIf(probe.NewError(e), "Cannot get service status.")
 
-	// Construct the backend status
-	storageInfo := backendStatus{
-		Total: serverInfo.StorageInfo.Total,
-		Free:  serverInfo.StorageInfo.Free,
-	}
-	if serverInfo.StorageInfo.Backend.Type == madmin.Erasure {
-		storageInfo.Backend = xlBackend{
-			Type:         erasureType,
-			OnlineDisks:  serverInfo.StorageInfo.Backend.OnlineDisks,
-			OfflineDisks: serverInfo.StorageInfo.Backend.OfflineDisks,
+	for _, serverInfo := range serversInfo {
```

Comment:
Here is the diff 

```diff
 
 // String colorized service status message.
 func (u infoMessage) String() (msg string) {
-       defer func() {
-               msg = console.Colorize("Info", msg)
-       }()
        // When service is offline
        if !u.Service {
-               msg = "The server is offline."
+               msg += fmt.Sprintf("%s %s\n", console.Colorize("InfoFail", "●"), u.ServerInfo.Addr)
+               msg += fmt.Sprintf("  Uptime : server is %s\n", console.Colorize("InfoFail", "offline"))
                return
        }
+       msg += fmt.Sprintf("%s %s\n", console.Colorize("Info", "●"), u.ServerInfo.Addr)
+       msg = console.Colorize("Info", msg)
        msg += fmt.Sprintf("  Version : %s\n", u.ServerInfo.Properties.Version)
-       msg += fmt.Sprintf("   Uptime : %s\n", timeDurationToHumanizedDuration(u.ServerInfo.Properties.Uptime))
+       msg += fmt.Sprintf("   Uptime : %s since %s\n", console.Colorize("Info", "online"),
+               humanize.Time(time.Now().UTC().Add(-u.ServerInfo.Properties.Uptime)))
        msg += fmt.Sprintf("   Region : %s\n", u.ServerInfo.Properties.Region)
```

Observe the usage of `humanize.Time()` which is much more human readable.